.github/workflows/image-release.yaml

```diff
@@ -25,3 +25,32 @@ jobs:
       - name: Push 
         run: |
               make docker-push
+      - name: Build & Pushing cloudshell image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile-webtty
```

Comment:
You should change the Dockerfile to install the multiple platform ttyd (arm64 and amd64)

Reply:
fixed. linux command `arch` seems helps a lot !